server-edge.js

```diff
@@ -44,7 +44,8 @@ var helmetOptions = {
       },
     },
     frameguard: { action: 'SAMEORIGIN' },
-    crossOriginEmbedderPolicy: false
+    crossOriginEmbedderPolicy: false,
+    strictTransportSecurity: false
```

Comment:
this change doesn't seem related to the PR. Is it intentional? Does this "help/fix" the http/https problems we've had?

Reply:
Nope good catch, this must have snuck in when I was troubleshooting that issue. I will revert